### PR TITLE
Test 

### LINKED ISSUE #296: [C4GT] Network-Score: Add tests for 'MessageIdAlreadyExists'
## Description
Subtask under : https://github.com/cord-network/community/issues/7

New testcase can be tested under `cargo test -p pallet-network-score` after adding it in the code.

## Goals

- [ ] Add tests for `MessageIdAlreadyExists` for pallet/network-score

## Expected Outcome
- [ ] Test should assert for `MessageIdAlreadyExists` being returned properly in all the possible calls.
- [ ] If there are more than one function returning this error code, all of them should be validated either as one test case or as multiple test cases.
       
## Acceptance Criteria
NA

## Implementation Details
Look at other test cases, and add a test case for the same.
Ex: In pallets/network-membership check method `test_duplicate_member_request` which validates error code `MembershipAlreadyAcquired`

## Mockups / Wireframes
NA

---- 

## Product Name
CORD

## Organization Name
Dhiway

## Domain
Blockchain

## Tech Skills Needed
Rust

## Mentor(s)
@amarts 

## Complexity
[Low]

## Category
[Test]

## Sub Category
[Beginner friendly]

### PR DESCRIPTION
**Pull request by zeel991**
- fixes https://github.com/dhiway/cord/issues/296
- @vatsa287 , I have added the test

### CODE REVIEW

File: desktop.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ViewState]
```
Comment (vatsa287) [inline desktop.ini:1]: Not required.

File: node/cli/tests/purge_chain_works.rs [hunk 2]
```diff
@@ -16,10 +16,10 @@
 // You should have received a copy of the GNU General Public License
 // along with CORD. If not, see <https://www.gnu.org/licenses/>.
 
+#[allow(unused_imports)]
```
Comment (vatsa287) [inline node/cli/tests/purge_chain_works.rs]: Keep it default for now. Can remove this change.

File: node/cli/tests/state_pruning_works.rs [hunk 3]
```diff
@@ -13,8 +13,9 @@
 
 // You should have received a copy of the GNU General Public License
 // along with CORD. If not, see <https://www.gnu.org/licenses/>.
+#[allow(unused_imports)]
 use tempfile::tempdir;
-
+#[allow(unused_imports)]
```
Comment (vatsa287) [inline node/cli/tests/state_pruning_works.rs]: These changes as well.

File: node/testing/src/bench.rs [hunk 4]
```diff
@@ -95,9 +95,9 @@ pub fn drop_system_cache() {
 		return;
 	}
 
-	std::process::Command::new("sync")
-		.output()
-		.expect("Failed to execute system cache clear");
+	// std::process::Command::new("sync")
```
Comment (vatsa287) [inline node/testing/src/bench.rs]: Remove this change too.

**Review by vatsa287 — CHANGES_REQUESTED** [review]

File: node/testing/src/bench.rs (hunk 4 above)
Comment (zeel991) [reply node/testing/src/bench.rs]: I'll remove all the not required changes ,

File: desktop.ini (hunk 1 above)
Comment (vatsa287) [reply desktop.ini:1]: @zeel991 This file should not be part of PR. Please remove this.

Comment (vatsa287) [reply desktop.ini:1]: I will recommend to do below:
- Copy the changes made in file pallets/network-score/src/tests.rs
- Remove all other changes by `git reset --hard HEAD~1`
- Apply the copied changes to same location.
- Add that file with `git add .`
- Amend the commit with `git commit --amend`
- Push to same branch, so we have a clean PR with only tests.rs changes.

Comment (zeel991) [reply desktop.ini:1]: I deeply apologize for all this trouble , Thank you so much for guiding me !
I will follow these steps, and push it to the branch ,Thanks again .

**Pull request closed without merging**